create-environment/action.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'create-environment'
+description: 'Power Platform Admin Create Environment'
+inputs:
+  environment-url:
+    description: 'URL of Power Platform environment to connect with; e.g. "https://test-env.crm.dynamics.com"'
+    required: true
+
+  user-name:
+    description: 'Power Platform user name, e.g. myname@my-org.onmicrosoft.com'
+    required: true
+
+  password-secret:
+    description: 'Power Platform password. Do NOT checkin password, instead create a secret and reference it here with: see: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#using-encrypted-secrets-in-a-workflow'
+    required: true
+
+  name:
+    description: 'The name of the environment you are creating.'
+    required: true
+
+  region:
+    description: 'The name of the region for your environment. Default is 'unitedstates'.'
```

Comment:
you can also specify default values as default: 'unitedstates'

Reply:
not just a nit: make sure the action parameter defaults here agree with what index.ts does. This is indeed a bit of double book keeping; action.yaml also serves as the public documentation of the action